.github/workflows/ci_build_ios.yml

```diff
@@ -0,0 +1,63 @@
+name: Build iOS library
+
+on:
+  workflow_call:
+    outputs:
+      artifact-dir-base:
+        description: The base name of artifact directory (without the target suffix)
+        value: ${{ jobs.build-ios-libs.outputs.artifact-dir-base }}
+
+jobs:
+  build-ios-matrix:
```

Comment:
Seems quite a bit of code to avoid to write the two targets explicitly in the other job.
Also the targets we have in the env are the env we want to build locally, they are not necessary the same we want here. For sure for android we want more targets here then in the env.

Reply:
I had the idea to treat the `.env` as a single source of truth and avoid hardcoded values in the CI.

> Also the targets we have in the env are the env we want to build locally, they are not necessary the same we want here. For sure for android we want more targets here then in the env.

Yes, but could define an extra key value pair for it. i will remove it

Reply:
done